src/routes/api/index.js

```diff
@@ -14,6 +15,7 @@ router.use('/api/v1/auth', userRoutes);
 
 // set up auth routes v1
 router.use('/api/v1/auth', userRoutes);
+router.use('/api/v1/trip', tripsRouter);
```

Comment:
```suggestion
router.use('/api/v1/trips', tripsRouter);
```
It's recommended to adhere [RESTful standards](https://restfulapi.net/resource-naming/)
that's to say; use plural for naming resources

Reply:
@blaise82 What did you decide about resources naming?

Reply:
@hakoemmy  done

Reply:
@blaise82  that's great